x/gov/keeper/common_test.go

```diff
@@ -80,30 +105,23 @@ func setupGovKeeper(t *testing.T) (
 
 	// gomock initializations
 	ctrl := gomock.NewController(t)
-	acctKeeper := govtestutil.NewMockAccountKeeper(ctrl)
-	bankKeeper := govtestutil.NewMockBankKeeper(ctrl)
-	stakingKeeper := govtestutil.NewMockStakingKeeper(ctrl)
-	distributionKeeper := govtestutil.NewMockDistributionKeeper(ctrl)
-
-	acctKeeper.EXPECT().GetModuleAddress(types.ModuleName).Return(govAcct).AnyTimes()
-	acctKeeper.EXPECT().GetModuleAddress(disttypes.ModuleName).Return(distAcct).AnyTimes()
-	acctKeeper.EXPECT().GetModuleAccount(gomock.Any(), types.ModuleName).Return(authtypes.NewEmptyModuleAccount(types.ModuleName)).AnyTimes()
-	acctKeeper.EXPECT().AddressCodec().Return(address.NewBech32Codec("cosmos")).AnyTimes()
-
-	trackMockBalances(bankKeeper, distributionKeeper)
-	stakingKeeper.EXPECT().TokensFromConsensusPower(ctx, gomock.Any()).DoAndReturn(func(ctx sdk.Context, power int64) math.Int {
-		return sdk.TokensFromConsensusPower(power, math.NewIntFromUint64(1000000))
-	}).AnyTimes()
-
-	stakingKeeper.EXPECT().BondDenom(ctx).Return("stake", nil).AnyTimes()
-	stakingKeeper.EXPECT().IterateBondedValidatorsByPower(gomock.Any(), gomock.Any()).AnyTimes()
-	stakingKeeper.EXPECT().IterateDelegations(gomock.Any(), gomock.Any(), gomock.Any()).AnyTimes()
```

Comment:
Those preset expectations annoyed me because then I can't change them for the different test cases. For that reason I had to change `setupGovKeeper` so it's more customisable.